### PR TITLE
Implement Project Card

### LINKED ISSUE #10: Implement Project Card
image
image

* Implement Project card
* Considering our design has it flipped on every other one, a prop to determine direction may be helpful here

### PR DESCRIPTION
**Pull request by houstongunter**
- Added two project card components
- One component has the text left of the image
- Other component has the text right of the image
- Closes #10 
Image 7-5-24 at 12 02 PM

### CODE REVIEW

File: src/components/landing/ProjectCardLeft.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Image from "next/image";
+
+const ProjectCardLeft = ({ projectName, description }) => {
+  return (
+    <div className="flex flex-col-reverse md:flex-row items-center md:items-start">
+      {/* Text below the image on small screens */}
```
Comment (khandrew1) [inline src/components/landing/ProjectCardLeft.js]: remove all comments

File: src/components/landing/ProjectCardLeft.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
+        />
+        <div className="absolute inset-0 flex items-center justify-center">
+          <p
+            className="text-ai-blue-500 text-center"
+            style={{ fontSize: "clamp(1rem, 5vw, 2rem)" }}
```
Comment (khandrew1) [inline src/components/landing/ProjectCardLeft.js]: no normal css, you shouldnt even need to do something this complicated?

File: src/components/landing/ProjectCardRight.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
+        />
+        <div className="absolute inset-0 flex items-center justify-center">
+          <p
+            className="text-ai-blue-500 text-center"
+            style={{ fontSize: "clamp(1rem, 5vw, 2rem)" }}
```
Comment (khandrew1) [inline src/components/landing/ProjectCardRight.js]: no normal css again

File: src/components/landing/ProjectCardRight.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <div className="flex flex-col md:flex-row items-center md:items-start">
+      {/* Image container */}
+      <div className="relative w-full md:w-auto">
+        <Image
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
```
Comment (khandrew1) [inline src/components/landing/ProjectCardRight.js]: sorry for being confusing on this in the beginning, please import `projectCard.webp` on top like
`import projectCard from "@/public/projectCard.webp"` and then use that as the source and remove width and height

File: src/components/landing/ProjectCardLeft.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+      {/* Image container */}
+      <div className="relative w-full md:w-auto">
+        <Image
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
+        />
```
Comment (khandrew1) [inline src/components/landing/ProjectCardLeft.js]: sorry for being confusing on this in the beginning, please import `projectCard.webp` on top like
`import projectCard from "@/public/projectCard.webp"` and then use that as the source and remove width and height

**Comment by khandrew1** [comment]
Component files should be using `.jsx` file extension, also should be in the folder `components/projects`

File: src/app/page.js [hunk 6]
```diff
@@ -1,7 +1,20 @@
-import OfferingCards from "@/components/landing/OfferingCards";
+import React from "react";
```
Comment (khandrew1) [inline src/app/page.js]: remove react imports from every file

File: src/app/page.js [hunk 7]
```diff
@@ -1,7 +1,20 @@
...
+import ProjectCardRight from "@/components/projects/ProjectCardRight";
+import ProjectCardLeft from "@/components/projects/ProjectCardLeft";
 
 const Home = () => {
```
Comment (khandrew1) [inline src/app/page.js]: change this back to only offering cards, put these project cards on /projects

File: src/app/projects/page.js [hunk 8]
```diff
@@ -1,3 +1,5 @@
+import React from "react";
+
 const Page = () => {
   return <>Projects</>;
```
Comment (khandrew1) [inline src/app/projects/page.js]: put project card components on this page

File: src/components/projects/ProjectCardLeft.jsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+        </p>
+      </div>
+
+      {/* Image container */}
+      <div className="relative w-full md:w-auto">
+        <Image
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
```
Comment (khandrew1) [inline src/components/projects/ProjectCardLeft.jsx]: import image statically on top, `import projectCard from @/public/projectCard.webp` then use that as the src, `src={projectCard}` and remove static heights and width

**Review by khandrew1 — CHANGES_REQUESTED** [review]

**Comment by houstongunter** [comment]
I don't know why one of the tests failed and I don't know why it's saying I added the offering cards files and changed the location of information.jsx I swear I didn't touch those

**Comment by khandrew1** [comment]
don't worry smth messed up within the directories

File: src/components/projects/OfferingCards.js [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+      "Challenge yourself and team up with peers in our AI competitions. A chance to put theory into practice.",
+    icon: <GiPodium className="text-5xl text-ai-blue-500" />,
+  },
+  {
+    title: "AI NEWS AND OPPORTUNITIES",
+    description:
+      "Stay informed with the most recent developments, breakthroughs, and job opportunities in the AI realm.",
+    icon: <LiaNewspaperSolid className="text-5xl text-ai-blue-500" />,
+  },
+];
```
Comment (khandrew1) [inline src/components/projects/OfferingCards.js]: move this to a data file

Comment (khandrew1) [reply src/components/projects/OfferingCards.js]: also lowercase variables for data vars

**Review by khandrew1 — CHANGES_REQUESTED** [review]

**Review by khandrew1 — APPROVED** [review]

**Pull request merged**